api/config/cors.php

```diff
@@ -3,66 +3,39 @@
 return [
 
     /*
-     * A cors profile determines which origins, methods, headers are allowed for
-     * a given requests. The `DefaultProfile` reads its configuration from this
-     * config file.
-     *
-     * You can easily create your own cors profile.
-     * More info: https://github.com/spatie/laravel-cors/#creating-your-own-cors-profile
-     */
-    'cors_profile' => Spatie\Cors\CorsProfile\DefaultProfile::class,
-
-    /*
-     * This configuration is used by `DefaultProfile`.
-     */
-    'default_profile' => [
-
-        'allow_credentials' => true,
+    |--------------------------------------------------------------------------
+    | Cross-Origin Resource Sharing (CORS) Configuration
+    |--------------------------------------------------------------------------
+    |
+    | Here you may configure your settings for cross-origin resource sharing
+    | or "CORS". This determines what cross-origin operations may execute
+    | in web browsers. You are free to adjust these settings as needed.
+    |
+    | To learn more: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+    |
+    */
+
+    'paths' => ['*'],
+
+    'allowed_methods' => ['*'],
+
+    'allowed_origins' => [
+        'https://localhost:8080',
```

Comment:
@szainmehdi do we need localhost:8080 listed here as our domain structure for frontend requires the site to run off of nawhas.test:8080, maybe localhost:8080 should be replaced with nawhas.test:8080 ?

Reply:
I left these there for the sake of preserving defaults set by Laravel.